build-tests/localization-plugin-test-03/webpack.config.js

```diff
@@ -22,9 +19,11 @@ function resolveMissingString(localeNames, localizedResourcePath) {
   const result = {};
   for (const localeName of localeNames) {
     const expectedCombinedStringsPath = path.resolve(__dirname, 'localization', localeName, 'combinedStringsData.json');
-    if (FileSystem.exists(expectedCombinedStringsPath)) {
+    try {
       const loadedCombinedStringsPath = JsonFile.load(expectedCombinedStringsPath);
       result[localeName] = loadedCombinedStringsPath[contextRelativePath];
+    } catch (e) {
+      // If reading the file throws, it probably doesn't exist
```

Comment:
It's safest to do:
```ts
if (e.code !== 'ENOENT' && e.code !== 'ENOTDIR') {
    // File exists, but reading failed.
    throw e;
}
```

Reply:
Kinda nasty to put that everywhere we read a file.